app/src/main/res/values/strings_settings.xml

```diff
@@ -86,59 +86,60 @@
 
     <string name="gesture_action_default">None</string>
     <string name="gesture_action_default_value" translatable="false">none</string>
-    <string name="gesture_action_widget">Open Widgetspace</string>
+    <string name="gesture_action_widget">Open widget-space</string>
     <string name="gesture_action_handler">Use gesture handler\'s action</string>
-    <string name="gesture_action_app">Launch application</string>
+    <string name="gesture_action_app">Launch app</string>
     <string name="gesture_action_status">Expand notification bar</string>
     <string name="gesture_action_panel">Expand quick settings panel</string>
     <string name="gesture_action_list">Open app list</string>
 
-    <string name="gesture_action_app_dialog_title">Select application</string>
+    <string name="gesture_action_app_dialog_title">Select app</string>
 
-    <string name="pref_title_app_list_order">App list order</string>
+    <string name="pref_title_app_list_order">Order</string>
     <string name="app_list_order_normal">A to Z</string>
     <string name="app_list_order_inverse">Z to A</string>
-
+    <!-- Translate this as "An always open app list -->
     <string name="static_app_list">Always open app list</string>
```

Comment:
This now doesn't match "static favorite panel".
Here it doesn't matter what the translation is, because either it is always opened (when starting launcher),
or it is just always open.
"Always open app list",
or
"Expanded app list"
are alternatives

Reply:
I think the former fits the English wording better. It is never stuck in an open state (as the original summary states, it can be closed), but it is always open when the user first starts the launcher.

Remember, this option is further described in the summary. If this does sound vague, the translator must make the best use of the summary to describe it.